Doing some housekeeping. (#4027)
flutter/website: src/docs/development/add-to-app/android/project-setup.md

{% include app-figure.md image="development/add-to-app/android/project-setup/ide-wizard.png" %}

The Android Studio plugin automatically configures your
Android project to add your Flutter module as a dependency,
and your app is ready to build.

{{site.alert.note}}
  To see the changes that were automatically made to your
  Android project by the IDE plugin, consider using
  source control for your Android project before performing
  any steps. A local diff shows the changes.
{{site.alert.end}}

{{site.alert.tip}}
  By default, your project's Project pane is probably
  showing the 'Android' view. If you can't see your new
  Flutter files in the Project pane, ensure that
  your Project pane is set to display 'Project Files',
  which shows all files without filtering.
{{site.alert.end}}

Your app now includes the Flutter module as a dependency.

> You can jump to the [API usage documentations][]

Reviewer: ```suggestion
You can jump to the [API usage documentation][]
```

                └── ...
```

To depend on the AAR, the host app must be able
to find these files.

To do that, edit `app/build.gradle` in your host app

> such as it includes the local repository and the dependency:

Reviewer: ```suggestion
so that it includes the local repository and the dependency:
```

?

```

{{site.alert.important}}
  If you're located in China, use a mirror site such as
  `https://[a mirror site]/download.flutter.io` rather than the
  `storage.googleapis.com` domain directly. See our
  [Using Flutter in China][] page for information on mirrors.
{{site.alert.end}}

{{site.alert.tip}}
  You can also build an AAR for your Flutter module in Android Studio using
  the `Build > Flutter > Build AAR` menu.

{% include app-figure.md image="development/add-to-app/android/project-setup/ide-build-aar.png" %}
{{site.alert.end}}

  Your app now includes the Flutter module as a dependency.
  You can follow the next steps in the

>   [API usage documentation][].

Reviewer: This links to "Adding a Flutter screen to an Android app." Is "API usage documentation" the best description we can manage? (I think I expected it'd go to generated API docs.)
